ova/generate_ova.sh

```diff
@@ -85,6 +86,10 @@ function check_version() {
   fi
 }
 
+function generate_checksum() {
+    shasum  -a512 "${DESTINATION}/${ova_name}" > "${DESTINATION}/${ova_name}.sha512"
```

Comment:
There's an extra whitespace in `shasum -a512`

Reply:
@DFolchA

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/5a72389d9ddbedad13d02074112ad463a9d9e133